### PR TITLE
Add extra option to make zoom require a modifier key

### PR DESCRIPTION
**Pull request by ZachHaber**
An extra functionality that I set up internally which allows for preventing zoom without a specified modifier.
I honestly have no idea at this point where I found the approach to do this anymore

### CODE REVIEW

**Comment by Wokstym** [comment]
Maybe we could consider adding a storybook where we could test this scenario? (I am not very familiar with this approach so just a thought)

**Comment by ZachHaber** [comment]
@Wokstym, that's part of my other option of fixing the storybook types, because that way this prop will be available on the UI, though I could also just set it up so this one has a dedicated story too, since it's a fairly useful feature.

File: src/index.tsx [hunk 1]
```diff
@@ -180,6 +184,27 @@ const VisGraph = forwardRef<
     };
   }, [events, network]);
 
+  useEffect(() => {
+    if (!zoomKey || !network) {
+      return () => {};
```
Comment (WikKam) [inline src/index.tsx:185]: why not just `return`?

Comment (ZachHaber) [reply src/index.tsx:185]: There's an eslint rule that was on the previous project that's not on here: consistent return type, which usually requires ugly disable comments or just returning a no-op function

**Review by Wokstym — APPROVED** [review]
LGTM

**Pull request merged**